src/audio/module_adapter/module/volume/volume.c

```diff
@@ -1432,7 +1432,7 @@ static struct comp_dev *volume_new(const struct comp_driver *drv,
 
 	mod = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM, sizeof(*mod));
 	if (!mod) {
-		comp_err(dev, "module_adapter_new(), failed to allocate memory for module");
+		comp_err(dev, "volume_new(), failed to allocate memory for module");
```

Comment:
May need futher change to : volume_new(), failed to allocate memory for volume

Reply:
I'm inclined to leave it as is since its allocating for `processing_module` which means its technically correct